app/javascript/gobierto_TEMP/webapp/components/WidgetIndicatorIndividual.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="graph-individual">
+    <div class="graph-individual__barchart">
+      <div
+        class="graph-individual__barchart--active"
+        :style="{ width: `${percentage}%` }"
+      />
+    </div>
+    <div class="graph-individual__title">
+      {{ title }}
+    </div>
+    <div class="graph-individual__metric--container">
+      <template v-for="{ metric, value } in metrics">
+        <div
+          :key="metric"
+          class="graph-individual__metric"
+        >
+          <div class="graph-individual__metric--title">
+            {{ metric }}
+          </div>
+          <div class="graph-individual__metric--value">
+            {{ value }}
+          </div>
+        </div>
+      </template>
+    </div>
+    <div class="graph-individual__date">
+      {{ dateLabel }} {{ year }}
+    </div>
+  </div>
+</template>
+
+<script>
+import { date, percent } from "lib/shared"
+
+export default {
+  name: "WidgetIndicatorIndividual",
+  props: {
+    title: {
+      type: String,
+      default: null
+    },
+    values: {
+      type: Array,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      percentage: 0,
+      year: null,
+      metrics: [],
+      dateLabel: "Fecha: ",
```

Comment:
Maybe the labels should be provided by the API in some way? Not to change now, but to think on it

Reply:
But you're suggesting so for ALL labels or specific ones? (like those depending of the data)

Reply:
well, I'm thinking again and the types of attributes depend on the widget so for the moment I think they can live there